core/io/resource_format_binary.cpp

```diff
@@ -1785,7 +1785,7 @@ Error ResourceFormatSaverBinaryInstance::save(const String &p_path, const RES &p
 	f->store_32(FORMAT_VERSION);
 
 	if (f->get_error() != OK && f->get_error() != ERR_FILE_EOF) {
-		f->close();
+		memdelete(f);
```

Comment:
`f->close();` should still be kept, it's important for the OS that we release the file handle properly.

Reply:
Done